source/managing_permissions.md

```diff
@@ -16,22 +16,22 @@ to collaborate on your projects.
 
 ## Requiring passwords for accessing Enketo web forms
 
-By default, KoboToolbox only requires a username and password for accessing data
-and managing your projects. When a project has been deployed, the Enketo web
-form can be accessed by anyone who knows its URL. In this case, any individual
-with access to the form URL can send submissions to the project.
+By default, KoboToolbox requires a username and password for accessing data,
+ managing your projects, and submitting data. When a new project has been deployed, the Enketo web
+form will require authentication for an individual to be able to access the form and send submissions to the project.
 
-In some cases, you may wish to limit data entry to require that individuals be
+In some cases, you may wish to enable data entry from anyone who knows the Enketo web form URL. In this case, any individual with access to the form URL can send submissions to the project, without them being required to be
 logged into their KoboToolbox user accounts.
 
-- Go to **ACCOUNT SETTINGS** inside your KoboToolbox account
-- Check "Require authentication to see forms and submit data"
+- Go to **Collect data** inside your KoboToolbox account
+- Use the toggle button to enable "Allow submissions to this form without a username and password"
```

Comment:
- Below the Collect Data section, use the toggle button to enable "Allow submissions to this form without a username and password"